server/views/pages/appointments/create-and-edit/extra-information.njk

```diff
@@ -20,18 +20,50 @@
+                    {% endif %}
         </h1>
         <p class="govuk-body" data-qa="first-paragraph">
-            Add or edit any important information for who’s attending about how to prepare for their appointment. Note that changes will not appear on any movement slips that have already been printed.
+        {% if session.appointmentJourney.category.code == 'VLB' %}
+                         Enter the court the booking is for, plus:
+                         <ul>
+                             <li class='govuk-list govuk-list--bullet'> details of any briefings before or after the hearing </li>
+                             <li class='govuk-list govuk-list--bullet'> other important information for who's attending about how to prepare for their appointment </li>
+                         </ul>
+                       <p> This will be printed on the prisioner's movement slip. </p>
+                    {% else %}
+                        <p class="govuk-body" data-qa="first-paragraph">
+                                Add or edit any important information for who’s attending about how to prepare for their appointment. Note that changes will not appear on any movement slips that have already been printed.
+                        </p>
+                    {% endif %}
+
         </p>
       {% else %}
         <h1 class="govuk-heading-l">
-            <label class="govuk-label govuk-label--l" for="extraInformation">Add extra information (optional)</label>
+            {% if session.appointmentJourney.category.code == 'VLB' %}
+                 <label class="govuk-label govuk-label--l" for="extraInformation">Enter the court name and any extra information</label>
+            {% else %}
+                <label class="govuk-label govuk-label--l" for="extraInformation">Add extra information (optional)</label>
+            {% endif %}
         </h1>
+
         <p class="govuk-body" data-qa="first-paragraph">
-            Add any important information for who’s attending about how to prepare for their appointment. It will be printed on their movement slip.
-        </p>
+            {% if session.appointmentJourney.category.code == 'VLB' %}
+                 Enter the court the booking is for, plus:
+                 <ul>
+                     <li class='govuk-list govuk-list--bullet'> details of any briefings before or after the hearing </li>
+                     <li class='govuk-list govuk-list--bullet'> other important information for who's attending about how to prepare for their appointment </li>
+                 </ul>
+               <p> This will be printed on the prisioner's movement slip. </p>
+            {% else %}
+                <p class="govuk-body" data-qa="first-paragraph">
+                        Add any important information for who’s attending about how to prepare for their appointment. It will be printed on their movement slip.
+                </p>
```

Comment:
I doubt it will cause issues, but I think you might be wrapping a `<p>` directly inside another `<p>` here, both with a `data-qa="first-paragraph" attribute.

Also, per the specs you're not suppose to nest a `p` or `ul` tag inside a `p` so I think you'd probably need to removing the tag on line 53 and put it around the text in the VLB conditional ("Enter the court the booking is for, plus:").

It's minor but it's one of those things HTML validators complain about and can confuse assistive technology.